book/internals.rst

```diff
@@ -238,8 +238,8 @@ add the following code at the beginning of your listener method::
 
 .. tip::
 
-    If you are not yet familiar with the Symfony EventDispatcher, read the
-    :doc:`EventDispatcher component documentation </components/event_dispatcher/introduction>`
+    If you are not yet familiar with the Symfony EventDispatcher component,
```

Comment:
Well, that's not really consistent with how we use component names otherwise (e.g. HttpFoundation, DependencyInjection, etc.).